tests/integration/targets/ome_application_certificate/vars/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Move all this to inventory.networking and don't provide any values

Reply:
done